src/lib/djerba/render/render.py

```diff
@@ -7,6 +7,9 @@
 import os
 import pdfkit
 import traceback
+import requests 
```

Comment:
Why are `requests` and `configparser` imported? Please remove if not being used.